common/utils/filters.ts

```diff
@@ -115,3 +128,21 @@ export const getSelectedFilterColor = (form: HTMLFormElement): string => {
       : imagesColorValue;
   return imagesColor;
 };
+
+export const sortAggregationBucket = (
+  filter: CatalogueAggregationBucket[],
+  sortBy?: 'alphabetical' | 'count'
+): CatalogueAggregationBucket[] => {
+  let sortedfilter;
+  switch (sortBy) {
+    case 'count':
+      sortedfilter = filter.sort(sortByCount);
+      break;
+    case 'alphabetical':
+      sortedfilter = filter.sort(sortLabelByAlphabeticalOrder);
+      break;
+    default:
+      sortedfilter = filter;
+  }
+  return sortedfilter;
+};
```

Comment:
You can just return the value you need in the case statement without introducing variables.

```ts
switch (sortBy) {
    case 'count':
      return filter.sort(sortByCount);
      break;
    case 'alphabetical':
      return filter.sort(sortLabelByAlphabeticalOrder);
      break;
    default:
      return filter;
  }
```